### PR TITLE
docs: add `auth/a/*` to except list

### PR DESCRIPTION
**Pull request by datamweb**
See: #487

### CODE REVIEW

**Review by MGatner — COMMENTED** [review]
This worries me a little. If someone makes their own routes we might accidentally exclude them: authorization, authors, authentic_antiques

Are there so many routes we cannot specify them individually?

**Comment by kenjis** [comment]
```
+--------+-------------------------+--------------------------------------------------------------------+----------------+---------------+
| Method | Route                   | Handler                                                            | Before Filters | After Filters |
+--------+-------------------------+--------------------------------------------------------------------+----------------+---------------+
| GET    | /                       | \App\Controllers\Home::index                                       |                | toolbar       |
| GET    | register                | \CodeIgniter\Shield\Controllers\RegisterController::registerView   |                | toolbar       |
| GET    | login                   | \CodeIgniter\Shield\Controllers\LoginController::loginView         |                | toolbar       |
| GET    | login/magic-link        | \CodeIgniter\Shield\Controllers\MagicLinkController::loginView     |                | toolbar       |
| GET    | login/verify-magic-link | \CodeIgniter\Shield\Controllers\MagicLinkController::verify        |                | toolbar       |
| GET    | logout                  | \CodeIgniter\Shield\Controllers\LoginController::logoutAction      |                | toolbar       |
| GET    | auth/a/show             | \CodeIgniter\Shield\Controllers\ActionController::show             |                | toolbar       |
| POST   | register                | \CodeIgniter\Shield\Controllers\RegisterController::registerAction |                | toolbar       |
| POST   | login                   | \CodeIgniter\Shield\Controllers\LoginController::loginAction       |                | toolbar       |
| POST   | login/magic-link        | \CodeIgniter\Shield\Controllers\MagicLinkController::loginAction   |                | toolbar       |
| POST   | auth/a/handle           | \CodeIgniter\Shield\Controllers\ActionController::handle           |                | toolbar       |
| POST   | auth/a/verify           | \CodeIgniter\Shield\Controllers\ActionController::verify           |                | toolbar       |
| CLI    | ci(.*)                  | \CodeIgniter\CLI\CommandRunner::index/$1                           |                |               |
+--------+-------------------------+--------------------------------------------------------------------+----------------+---------------+
```

**Comment by MGatner** [comment]
Oh that's nice! They are already `auth/a/*`

File: docs/install.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ If you want to limit all routes (e.g. `localhost:8080/admin`, `localhost:8080/pa
 public $globals = [
     'before' => [
         // ...
-        'session' => ['except' => ['login*', 'register*']],
+        'session' => ['except' => ['login*', 'register*', 'auth*']],
```
Comment (kenjis) [inline docs/install.md:155]:
```suggestion
        'session' => ['except' => ['login*', 'register', 'auth/a/*']],
```

Comment (datamweb) [reply docs/install.md]: @kenjis thanks.

**Review by kenjis — APPROVED** [review]

**Review by MGatner — APPROVED** [review]

**Comment by MGatner** [comment]
Merging this for release.

**Pull request merged**